src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,395 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * This class stores all duke.TaskList of the user.
+ *
+ * @author Deng Huaiyu(G12)
+ * @version CS2103T AY21/22 Semester 1
+ */
+public class TaskList {
+    private static ArrayList<Task> taskList = new ArrayList<>();
+    //for division
+    private static String ind = "    ";
+    //for sentences
+    private static String ind2 = "     ";
+    private static String div = ind + "____________________________________________________________";
+
+    /**
+     * Adds a new task according to instruction
+     *
+     * @param t input from the user
+     * @throws DukeException if the input is invalid
+     */
+    public void addTask(String t) throws DukeException{
+        String[] ss = t.split(" ");
+        switch (ss[0]) {
+            case("todo"):
```

Comment:
Not sure about the indentation for switch case. I believe case should be 4 spaces to the left.

Reply:
Thanks for the comment! From coding standard, the case clauses should not be indented.